src/main/java/duke/command/DoneCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.main.*;
```

Comment:
I think we should avoid the wildcard import as much as possible, but I understand that Intellij does it automatically. It is best we manually and explicitly list out the classes we are importing.